### ISSUE TITLE
Retraining Bodypix Models or Posenet

### ISSUE THREAD

**Issue by KushalHM** [google-coral/project-bodypix#31]
### Description

Hi,
I would like to retrain the posenet component of bodypix with some additional keypoint labelled images and also with some augmented images (images rotated at certain degree).

This is to enhance the capability of the posenet component to better detected inverted (fallen down) person or a person spotted at a particular angle relative to the camera view.

Is there any to retrain these models, like training script with pretrained model? If not is there any way to improve the accuracy of these models to better detect fallen down humans?

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Feature Request

### Operating System

Ubuntu

### Coral Device

Dev Board

### Programming Language

C++, Python 3.8

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
Hello @KushalHM Unfortunately, we don't have any scripts or support to train a bodypix model. Please check this [link]( https://www.tensorflow.org/lite/examples/pose_estimation/overview) and reach out to [tensorflow team](https://github.com/tensorflow/tensorflow) for any support. Thanks!

**Comment by KushalHM** [comment]
Thanks for the quick response. I will reach out to tensorflow team.
